.gitignore

```diff
@@ -12,6 +12,13 @@ __pycache__/
 instance/
 .webassets-cache
 
+# Heroku environmental variables
+.env
+
 # virtualenv
 .venv
 venv/
+env/
```

Comment:
Rather than add a duplicate virtualenv entry in gitignore, why not just re-init your virtual env? It should be safe to just do `rm -rf env` and `virtualenv venv`.

Reply:
Yeah, I actually just went ahead and stuck `.vscode` and `env/` in my `.git/info/exclude file` (which was a thing I didn't know existed until today 😍 ), so I'm all set for those two.

The heroku `.env` thing, though, I'm not sure on, so I'll wait on @karlcow .